src/components/Footer/FooterCookie/FooterCookie.tsx

```diff
@@ -13,19 +13,19 @@ export const FooterCookie = () => {
     <div
       className={`${
         isVisible ? '' : 'hidden'
-      } fixed mx-auto max-w-7xl p-6 border-4 rounded-md shadow-lg bg-cu-black-25 bottom-6 right-6 left-6 border-white`}
+      } fixed mx-auto max-w-7xl p-6 border-4 rounded-md shadow-lg bg-cu-black-25 bottom-6 right-6 left-6 border-white cu-cookie-notice`}
```

Comment:
@troychaplin Do we need this cu-* classes in cookie? These classes are specific to cutheme only used to hide the notice. Can we remove it?